src/Umbraco.Cms.Api.Management/Controllers/Dictionary/DeleteDictionaryController.cs

```diff
@@ -16,36 +16,20 @@ public DeleteDictionaryController(ILocalizationService localizationService, IBac
         _localizationService = localizationService;
         _backOfficeSecurityAccessor = backOfficeSecurityAccessor;
     }
-    /// <summary>
-    ///     Deletes a data type with a given ID
-    /// </summary>
-    /// <param name="key">The key of the dictionary item to delete</param>
-    /// <returns>
-    ///     <see cref="HttpResponseMessage" />
-    /// </returns>
+
     [HttpDelete("{key}")]
     [MapToApiVersion("1.0")]
     [ProducesResponseType(StatusCodes.Status200OK)]
-    [ProducesResponseType(typeof(NotFoundResult), StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
     public async Task<IActionResult> Delete(Guid key)
     {
-        IDictionaryItem? foundDictionary = _localizationService.GetDictionaryItemByKey(key.ToString());
-
-        if (foundDictionary == null)
+        IDictionaryItem? dictionaryItem = _localizationService.GetDictionaryItemById(key);
```

Comment:
Should the service layer expose a DeleteByKey instead of having the logic here?

Reply:
I've updated the service layer to use same pattern/methodology as Create and Update